### PR TITLE
RUN-11694-templates-file-added

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

**Comment by jasonnovichRunAI** [comment]
@omerbenedictrunai and @liohill please review. This is a new page based on a gap.

**Review by omerbenedictrunai — COMMENTED** [review]
I would expand a little bit the explanation (line 11) with an example. Currently it is very abstract, which would make it hard to the end user understand.

File: docs/admin/admin-ui-setup/templates.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Templates
+summary: This article describes the templates form. This form allows explains what a template is, and where it is used..
+authors:
+    - Jason Novich
+date: 2023-oct-22
+---
+
+A template is a pre-set configuration that is used help quickly configure assets in the system. Using a template allow you to submit a workload based on template which has aggregated all relevant building blocks for your workload.
```
Comment (jasonnovichRunAI) [inline docs/admin/admin-ui-setup/templates.md:9]: @omerbenedictrunai @liohill Please review the added text.
```suggestion
A template is a pre-set configuration that is used help quickly configure assets in the system. Using a template allows you to submit a workload which has aggregated all the relevant building blocks for your workload. This helps optimze workload submission, especially in cases where workloads may need the same or very similar settings.
```

Comment (liohill) [inline docs/admin/admin-ui-setup/templates.md:9]: This is not true: "A template is a pre-set configuration that is used help quickly configure assets in the system."

Comment (liohill) [reply docs/admin/admin-ui-setup/templates.md:9]:
```suggestion
A template is a pre-set configuration that is used to quickly submit workloads using existing assets.
A template can consist of all the setup a workload needs, thus allowing researchers to submit a workload in a single click.
```

File: docs/admin/admin-ui-setup/templates.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Creating Templates
+
+To create a template:
+
+1. In the left menu, press *Templates*, then press *New Template*.
+2. In the *Scope* pane, select a *Scope*.
+3. In the *Template name* pane, enter a name.
+4. In the *Environment* pane, select an environment from the list. Use the *Search environments* if you do not see your environment listed. Press *New environment* to add a new environment to the system. Press *More settings* to add an `Environment variable` or to edit the *Command* and *Arguments* field for the environment you selected.
```
Comment (liohill) [inline docs/admin/admin-ui-setup/templates.md:18]: we need to mention here that the environment is the only real mandatory asset to select in a template. In other words- the template is not obliged to state the data source or compute resource, but the environment is a must

Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/templates.md:18]: @liohill the template comes with a "default" environment variable. 
image

File: docs/admin/admin-ui-setup/templates.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/templates.md:9]:
```suggestion
A template is a pre-set configuration that is used to quickly configure and submit workloads using existing assets.
A template consists of all the assets a workload needs, allowing researchers to submit a workload in a single click, or make subtle adjustments to differentiate them from each other.
```

File: docs/admin/admin-ui-setup/templates.md (hunk 2 above)
Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/templates.md:18]: > we need to mention here that the environment is the only real mandatory asset to select in a template. In other words- the template is not obliged to state the data source or compute resource, but the environment is a must

@liohill The envorinment variable comes as default when you create the environment. See image:
image
So even if you do not choose to make changes to this, the field is there. You can add more than one env variable.

Maybe this:
```suggestion
4. In the *Environment* pane, select an environment from the list. Use the *Search environments* if you do not see your environment listed. Press *New environment* to add a new environment to the system. Press *More settings* to edit or add an `Environment variable`, or to edit the *Command* and *Arguments* field for the environment you selected.
```

**Review by jasonnovichRunAI — COMMENTED** [review]
@liohill Please see updated comments.

**Pull request merged**